frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -2445,6 +2445,18 @@ function ReaderFooter:onNetworkConnected()
 end
 ReaderFooter.onNetworkDisconnected = ReaderFooter.onNetworkConnected
 
+function ReaderFooter:onSwapPageTurnButtons()
+    if self.settings.page_turning_inverted then
+        self:maybeUpdateFooter()
+    end
+end
+
+function ReaderFooter:onToggleReadingOrder()
+    if self.settings.page_turning_inverted then
+        self:maybeUpdateFooter()
+    end
+end
+
```

Comment:
```suggestion
ReaderFooter.onToggleReadingOrder = ReaderFooter.onSwapPageTurnButtons
```

(Instead of duplicating the code ;)).

Reply:
that's fine, but why doesn't it work? what am I missing?

Reply:
this works

```lua
function ReaderFooter:onSwapPageTurnButtons()
    UIManager:scheduleIn(0.1, self.autoRefreshFooter)
end
ReaderFooter.onToggleReadingOrder = ReaderFooter.onSwapPageTurnButtons
```
kind of, because for some reason it won't update if you toggle the swipes from the menu, swapping page-turn button from menu works though. with gestures both work fine. And the two page-turn inversion that don't have an event also don't work